cli/sawtooth_cli/cluster.py

```diff
@@ -302,6 +301,10 @@ def do_cluster_start(args):
 
     save_state(state)
 
+    # Make sure the docker cluster network is up before starting nodes
+    if manage_type == 'docker':
+        node_controller.start_cluster_network()
+
```

Comment:
As discussed yesterday: I think this should be moved into `manage/sawtooth_manage/docker.py` Since, as you mentioned, this would cause a race condition where all the threads try to create a docker network, you will need to use a condition variable or something similar: https://en.wikipedia.org/wiki/Monitor_(synchronization)#Condition_variables